public/coffee/directives/asyncForm.coffee

```diff
@@ -33,6 +33,10 @@ define [
 							response.success = true
 							response.error = false
 
+							if onSuccessHandler = scope[attrs.onSuccess]
+								onSuccessHandler(data, status, headers, config)
```

Comment:
is this 'if' shorthand for

    onSuccessHandler = scope[attrs.onSuccess]
    if onSuccessHandler?
        dostuf()
   
if so its a touch unusual imo